pylxd/client.py

```diff
@@ -98,28 +101,35 @@ def netloc(self):
 
     def get(self, *args, **kwargs):
         """Perform an HTTP GET."""
+        kwargs['timeout'] = kwargs.get('timeout', self._timeout)
         response = self.session.get(
             self._api_endpoint, *args, **kwargs)
         self._assert_response(response, stream=kwargs.get('stream', False))
         return response
 
     def post(self, *args, **kwargs):
         """Perform an HTTP POST."""
-        response = self.session.post(self._api_endpoint, *args, **kwargs)
+        kwargs['timeout'] = kwargs.get('timeout', self._timeout)
+        response = self.session.post(
+            self._api_endpoint, *args, **kwargs)
```

Comment:
Sorry, but one final thing: this line `response = ...` all used to be on the same line, but went multi-line with the addition of the timeout parameter.  Now that's gone, please can it go back to one line?  Thanks!